src/Transform.cpp

```diff
@@ -240,6 +263,14 @@ void Transform::set_parent_dirty()
     needs_bounding_box_adjusting = true;
 }
 
+void Transform::set_children_dirty()
+{
+    for (auto&& child : children)
+    {
+        child->set_parent_dirty();
+    }
+}
+
```

Comment:
Function should be const.
`auto&&` is a mistake on my part, it should probably just be `auto const&`